src/components/ImportForm/SourceSection/SourceSection.tsx

```diff
@@ -162,7 +164,9 @@ const SourceSection: React.FC<SourceSectionProps> = () => {
           label={label}
           validated={!isValid && ValidatedOptions.error}
           isRequired
-          labelIcon={<HelpPopover bodyContent="Make sure the URL is correct." />}
+          labelIcon={
+            <HelpPopover bodyContent="Make sure the URL is correct. If the repository is private, provide authorization." />
```

Comment:
We do not have authorization options for private repo. This is not a valid help text.